<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DFT vs FFT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dftBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw DFT</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fftBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw FFT</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+F</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="plotHere">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>521</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clrBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+C</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="plotHere_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>521</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="errorBtn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Error</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+E</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
